servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/DefaultGrpcServiceContext.java

```diff
@@ -64,6 +65,11 @@ public GrpcExecutionContext executionContext() {
         return connectionContext.socketOption(option);
     }
 
+    @Override
+    public Protocol protocol() {
```

Comment:
Should we make `GrpcProtocol` public? There is no additional info (like version or something else) except the name.

Reply:
As gRPC is a layered application protocol, I think it is valuable to provide the underlying protocol (HTTP/1/2/3) information. We can either nest `Protocol`:

`GrpcProtocol#httpProtocol()` 

or just create top level constants: `GrpcProtocol#GRPC_OVER_HTTP_1`, `GrpcProtocol#GRPC_OVER_HTTP_2`, etc

I have a slight preference for top-level constants.